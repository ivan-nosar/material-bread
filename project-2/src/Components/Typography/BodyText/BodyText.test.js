import React from 'react';
import { BodyText } from '../../..';

import renderer from 'react-test-renderer';

test('BodyText Renders', () => {
  const tree = renderer.create(<BodyText />).toJSON();
  expect(tree).toMatchSnapshot();
});
