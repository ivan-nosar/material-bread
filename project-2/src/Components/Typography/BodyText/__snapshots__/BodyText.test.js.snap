// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BodyText Renders 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "rgba(0, 0, 0, 0.87)",
      },
      Object {
        "fontSize": 16,
        "letterSpacing": 0.5,
        "lineHeight": 16,
      },
      Object {
        "color": "rgba(0,0,0, 87)",
        "marginBottom": 0,
        "textAlign": "left",
      },
      undefined,
    ]
  }
/>
`;
